test/uk/gov/hmrc/customs/emailfrontend/views/CheckYourEmailSpec.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customs.emailfrontend.views
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.scalatest.matchers.must.Matchers.mustBe
+import play.api.Application
+import play.api.i18n.{Messages, MessagesApi}
+import play.api.mvc.AnyContentAsEmpty
+import play.api.test.FakeRequest
+import uk.gov.hmrc.customs.emailfrontend.config.AppConfig
+import uk.gov.hmrc.customs.emailfrontend.forms.Forms.confirmEmailForm
+import uk.gov.hmrc.customs.emailfrontend.utils.Utils.emptyString
+import uk.gov.hmrc.customs.emailfrontend.utils.{FakeIdentifierAgentAction, SpecBase}
+import uk.gov.hmrc.customs.emailfrontend.views.html.check_your_email
+import uk.gov.hmrc.customs.emailfrontend.controllers.routes
+
+class CheckYourEmailSpec extends SpecBase with ViewTestHelper {
```

Comment:
Remove SpecBase from here as ViewTestHelper already extends SpecBase. So it would become

class CheckYourEmailSpec extends ViewTestHelper

Same comment is applicable for other two test classes